Prokopishena.University.Personalized-Fitness-Coaching-Platform.Web/static/script/profile.js

```diff
@@ -0,0 +1,1077 @@
+                <p>Training Goals: ${client.training_goals}</p>
+            `;
+
+            clientDiv.appendChild(additionalInfoDiv);
+
+            const button = document.createElement('button');
+            button.textContent = 'All information';
+            button.addEventListener('click', toggleClientInfo);
+            clientDiv.appendChild(button);
+
+            // Створення кнопки для відкриття форми рекомендацій
+            const nutritionsButton = document.createElement('button');
+            nutritionsButton.textContent = 'Nutritions';
+            nutritionsButton.addEventListener('click', () => openNutritionsForm(client));
+            clientDiv.appendChild(nutritionsButton);
+
+            clientContainer.appendChild(clientDiv);
+        });
+    } catch (error) {
+        console.error('Error fetching clients for trainer:', error);
+    }
+};
+
+const openNutritionsForm = async (client) => {
+    const clientId = client.client_id;
+    const clientDiv = document.querySelector(`div[data-client-id="${clientId}"]`);
+    
+    if (clientDiv) {
+        // Перевіряємо, чи вже створено форму
+        if (clientDiv.querySelector('form')) {
+            console.log('Nutritions form already exists');
+            return;
+        }
+
+        // Створюємо форму
+        const form = document.createElement('form');
+        form.style.display = 'block'; // Показуємо форму при відкритті
+        form.innerHTML = `
+            <label for="recommendation_text">Enter Nutrition Recommendation:</label><br>
+            <textarea id="recommendation_text" name="recommendation_text" rows="4" cols="50"></textarea><br>
+            <button type="button">Send Recommendation</button>
+        `;
+        clientDiv.appendChild(form);
+
+        form.querySelector('button').addEventListener('click', async () => {
+            const recommendationText = form.querySelector('#recommendation_text').value;
+            const userId = localStorage.getItem('user_id');
+            const trainerId = await getTrainerId(userId);
+
+            try {
+                const response = await fetch('/nutrition-recommendations', {
+                    method: 'POST',
+                    headers: {
+                        'Content-Type': 'application/json'
+                    },
+                    body: JSON.stringify({
+                        client_id: clientId,
+                        trainer_id: trainerId,
+                        recommendation_text: recommendationText
+                    })
+                });
+
+                if (!response.ok) {
+                    console.error('Error sending recommendation');
+                    return;
+                }
+
+                form.querySelector('#recommendation_text').value = '';
+            } catch (error) {
+                console.error('Error sending recommendation:', error);
+            }
+        });
+    } else {
+        console.error('Client div not found');
+    }
+};

```

Comment:
Not sure if this Nutritions is working on the website. Maybe try to re-check & fix this.